execution/tvf/range.go

```diff
@@ -48,28 +51,45 @@ func (r *Range) Get(ctx context.Context, variables octosql.Variables, streamID *
 		return nil, nil, errors.Errorf("invalid range start point: %v", end)
 	}
 
+	currentIndexState := storage.NewValueState(storage.GetStateTransactionFromContext(ctx).WithPrefix(streamID.AsPrefix()).WithPrefix(currentIndexPrefix))
```

Comment:
Afaik we've always declared `tx := storage.GetStateTransactionFromContext(ctx)` separately, but I guess that's OK since we use that only for this value state